DEVELOP.md

```diff
@@ -73,6 +73,7 @@ gulp                    # Compile the JS and start the automatic
 To regenerate the `bundled.js` file without using the live reloader, run:
 
 ```
+gulp peg
```

Comment:
It takes longer than just running `gulp peg` and the extra work is not needed. Any reason do tell users  to use `gulp prod` instead? We should also put the `peg` task into the `default` task, obviating the need for this entirely, but adding more startup time each time you start `gulp`.